tests/README.md

```diff
@@ -66,7 +66,7 @@ $ virtualenv <name-of-virtualenv-folder>
 
 ##### (Optional) -- Select Python Version
 ```bash
-$ virtualenv -p /usr/bin/python2.7 <name-of-virtualenv-folder>
+$ python -m venv <name-of-virtualenv-folder>
```

Comment:
Minor nitpick:
Should probs be `python3` on most systems, no?

Reply:
Hopefully this will be changing this year, right? Ubuntu had this "python2" package which removes the "python" symlink so that it can be replaced by python3. I don't think it is very important, any educated reader know what are we referring to...

Reply:
Alright, good to go then :smiley_cat: